README.md

```diff
@@ -74,6 +74,20 @@ npx degit "rollup/rollup-starter-app" [NAME]
 
 The [Webpack example](./webpack/) was built from scratch using Webpack 4.x.
 
+## Visual Studio IntelliSense
+
+Calcite supports IntelliSense in Visual Studio Code. You can quickly add components and their attributes or properties while writing your app. There is also accompanying documentation to help you along the way.
```

Comment:
Should we add a reference to the [Visual Code reference guide](https://code.visualstudio.com/docs/editor/intellisense) for additional context if folks want to learn more about IntelliSense?

Reply:
> There is also accompanying documentation to help you along the way.

Is there a link or reference you were thinking would accompany this sentence?

Reply:
> > There is also accompanying documentation to help you along the way.
> 
> Is there a link or reference you were thinking would accompany this sentence?

😄 This is taken from the January 2023 summary, I thought the same thing when I first read it but then figured it must be talking about the documentation that pops up with the IntelliSense.

Reply:
> 😄 This is taken from the January 2023 summary, I thought the same thing when I first read it but then figured it must be talking about the documentation that pops up with the IntelliSense.

Kitty from the past had bold predictions. 😆 Maybe we can omit it here, or could this be expanded on? It makes me wonder what it means, and if its even needed in the explanation? Maybe the IntelliSense link is sufficient here?

Reply:
😂 So in the preview gif as you type you can see the info popping up for a split second, I believe thats what it's referencing and if so then it's valid. 

We could merge and tweak the last two sentences to something like "You can quickly add components and their attributes or properties while writing your app, with accompanying pop-up documentation to help you along the way."

Is that a bit more clear?

Reply:
Dig that approach 🧠. Maybe a bit of copyediting to it to keep it. Something like:

> Calcite supports IntelliSense in Visual Studio Code. You can quickly add components and their attributes or properties in the Visual Studio Code editor, where accompanying documentation will help you along the way.